test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -149,15 +150,15 @@ func perfScheduling(test testCase, b *testing.B) []DataItem {
 	}
 	defer nodePreparer.CleanupNodes()
 
-	total := 0
 	for _, p := range test.InitPods {
 		if err := createPods(setupNamespace, p, clientset); err != nil {
 			b.Fatal(err)
 		}
-		total += p.Num
-	}
-	if err := waitNumPodsScheduled(b, total, podInformer, setupNamespace); err != nil {
-		b.Fatal(err)
+		if !p.SkipWaitingUntilScheduled {
```

Comment:
nit: keep this outside loop.

I feel like it could open other test cases involving uni directional affinity.

Reply:
ok, how about we move the parameter outside the case then and have a single flag for all init pods, otherwise this is not really going to work well.

Reply:
oh I missed that the parameter is inside the case. We an leave it like this for now until we have a reason to change it.

Reply:
I think we should do it outside, we can't enforce which exact pods we are waiting for in the loop unless each case creates the pods in a different namespace.

Reply:
sg

Reply:
squashed